.openapi-generator/VERSION

```diff
@@ -0,0 +1 @@
+4.2.3-SNAPSHOT
```

Comment:
Hi @AntoineDao, is this version looking correct? Shouldn't it be 5.0 or 4.3.1?

Reply:
It's the version I got from using the fixed `datadog/apigentools:git-2b42a76` container 😀 It should stay fixed to this between builds from now on!